__tests__/app.test.js

```diff
@@ -37,65 +37,103 @@ describe("/api/categories", () => {
   });
 });
 
-
 describe("/api/reviews/:review_id", () => {
-    test("status 200, returns the correct review", ()=>{
-        return request(app)
-        .get ("/api/reviews/4")
-        .expect(200)
-        .then(({body})=>{
-          expect(body.review).toMatchObject({
-            review_id: 4,
-            title: 'Dolor reprehenderit',
-            category: 'social deduction',
-            designer: 'Gamey McGameface',
-            owner: 'mallionaire',
-            review_img_url: 'https://images.pexels.com/photos/278918/pexels-photo-278918.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260',
-            created_at: '2021-01-22T11:35:50.936Z',
-            votes: 7,
-            review_body: 'Consequat velit occaecat voluptate do. Dolor pariatur fugiat sint et proident ex do consequat est. Nisi minim laboris mollit cupidatat et adipisicing laborum do. Sint sit tempor officia pariatur duis ullamco labore ipsum nisi voluptate nulla eu veniam. Et do ad id dolore id cillum non non culpa. Cillum mollit dolor dolore excepteur aliquip. Cillum aliquip quis aute enim anim ex laborum officia. Aliqua magna elit reprehenderit Lorem elit non laboris irure qui aliquip ad proident. Qui enim mollit Lorem labore eiusmod'
-          })
-        })
-    })
-    test("status 400, if given a invalid review id", () =>{
-      return request(app)
-      .get ("/api/reviews/banana")
+  test("status 200, returns the correct review", () => {
```

Comment:
There are a lot of changes to end points that aren't under review, please if you are going to implement feedback do so on the branch of the PR, could be a formatting thing tho so no worries if youre not doing this